src/components/font_size_editor/font_size_editor.xml

```diff
@@ -0,0 +1,35 @@
+<templates>
+  <t t-name="o-spreadsheet-FontSizeEditor" owl="1">
+    <div class="o-dropdown o-font-size-editor" t-ref="FontSizeEditor">
+      <input
+        type="number"
+        min="1"
+        max="400"
+        class="o-font-size bg-transparent border-0"
+        t-on-keydown="onInputKeydown"
+        t-on-click.stop=""
+        t-on-focus.stop="onInputFocused"
+        t-att-value="currentFontSize"
+        t-on-change="setSizeFromInput"
+        t-ref="inputFontSize"
+      />
+      <span title="Font Size" t-on-click.stop="() => this.toggleFontList()">
```

Comment:
Same here, why `.stop` ?

Reply:
That was needed in the first implementation I made but it looks like after all the new changes it's not needed anymore